config/v1/types_cluster_version.go

```diff
@@ -215,6 +222,58 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionCapabilitiesDefault is a constant representing the
+// default policy for optional, core cluster components.
+type ClusterVersionCapabilitiesDefault string
+
+const (
+	// IncludeClusterVersionCapabilities indicates that optional,
+	// core cluster components should be included by default.
+	IncludeClusterVersionCapabilities = "Include"
+
+	// ExcludeClusterVersionCapabilities indicates that optional,
+	// core cluster components should be excluded by default.
+	ExcludeClusterVersionCapabilities = "Exclude"
+)
+
+// ClusterVersionCapabilitiesSpec configures the installation of
+// optional, core cluster components.
+// +k8s:deepcopy-gen=true
+type ClusterVersionCapabilitiesSpec struct {
+	// inclusionDefault declares the default policy for optional,
+	// core cluster components.
+
+	// +optional
+	InclusionDefault ClusterVersionCapabilitiesDefault `json:"inclusionDefault,omitempty"`
```

Comment:
The doc here is pretty rough, but you're probably trying to separate, "I want only the things installed in my 4.10 cluster on in my 4.11 cluster" from "I want the default capabilities in my cluster for whatever version I'm on except for these special cases".  I think the API could be better formed for that if it is the objective.